go-controller/pkg/testing/exec.go

```diff
@@ -139,6 +148,24 @@ func getExecutedCommandline(cmd string, args ...string) string {
 }
 
 func (f *FakeExec) Command(cmd string, args ...string) kexec.Cmd {
+	if f.receivedUnexpected {
```

Comment:
If we end up here we don't call `f.internalErrorDesc()`, should we not do that?

Reply:
Once the bool is raised, it's because an unexpected command was executed before, so all comes after is not reliable.
We return this dummy command so the execution goes on, but what the dev should care about and fix is only the first unexpected.